packages/catalogue/examples/example.js

```diff
@@ -0,0 +1,58 @@
+import { credentialsProvider, init as initAuth } from '@tidal-music/auth';
+
+import { catalogueClient, initCatalogueClient } from '../dist';
+
+async function runExample(clientId, clientSecret) {
+  await initAuth({
+    clientId,
+    clientSecret,
+    credentialsStorageKey: 'clientCredentials',
+  });
+
+  await initCatalogueClient(credentialsProvider);
+
+  const { data, error } = await catalogueClient.GET('/albums/{id}', {
+    params: {
+      path: { id: '251380836' },
+      query: { countryCode: 'no' }, // would be nice with a default value?
```

Comment:
+1 default value

Reply:
Unfortunately have not managed to get to get that trough 😢